src/yunohost/domain.py

```diff
@@ -41,24 +41,29 @@
 logger = getActionLogger('yunohost.domain')
 
 
-def domain_list():
+def domain_list(exclude_subdomains=False):
     """
     List domains
 
     Keyword argument:
         filter -- LDAP filter used to search
         offset -- Starting number for domain fetching
         limit -- Maximum number of domain fetched
```

Comment:
Unrelated, but these arguments have been removed here: https://github.com/YunoHost/yunohost/commit/56513bbdb121aa654cff9bbb28241addbf0fb7ab YunoHost 2.7.1, we should consider removing them from the comments :-)